pkg/cmd/node/node_diagnostics_cancel.go

```diff
@@ -0,0 +1,34 @@
+package node
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+func newCmdNodeDiagnosticsCancel(ctx api.Context) *cobra.Command {
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:   "cancel",
+		Short: "Cancel a running diagnostics job",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resp, err := diagnosticsClient().CancelBundle()
+			if err != nil {
+				return err
+			}
+
+			if jsonOutput {
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(resp)
+			}
+			fmt.Println(resp.Status)
```

Comment:
With the python CLI, is it printing the HTTP response status code in case of success? Do we still want to do this?

Reply:
Yes it does with the Python CLI:
```
➜  dcos-core-cli (DCOS_OSS-4697) ✔ dcos node diagnostics --cancel
Job is not running
```
`Job is not running` is the status of the response. I think it is useful as if a job is not running we have not canceled anything.